force-app/main/default/pages/STG_PanelDataImportAdvancedMapping.page

```diff
@@ -24,82 +42,88 @@
     </style>
 
     <div class="slds-scope stg-panel-data-import-advanced-mapping">
-        <apex:form id="form" styleClass="slds-m-around_x-large">
-            <c:STG_PageHeader sectionLabel="{!$Label.stgNavSystem}" pageLabel="{!$Label.bdiAdvancedMapping}" />
 
-            <c:UTIL_PageMessages escMsg="false" allowClose="false" id="messages" />
-            <c:UTIL_SingleToastMessage isRendered="{!NOT(isAdmin)}" message="{!$Label.stgDIFMNonAdminError}" />
```

Comment:
I like how we were showing a message, but not preventing anything else from loading 🙃